### ISSUE TITLE
Section 5.8: RTCP mux indication may change

### ISSUE THREAD

**Issue by hardie** [rtcweb-wg/jsep#403]
*  If RTCP mux is indicated, prepare to demux RTP and RTCP from
         the RTP ICE component, as specified in [RFC5761],
         Section 5.1.1.  If RTCP mux is not indicated, but was indicated
         in a previous description, this MUST result in an error.

I note that there can be difference here between an offer and an answer if the RTCP mux indication may have changed or not between these two description.

**Comment by juberti** [comment]
Agreed. If we offered rtcp-mux, but it wasn't accepted, the next offer would be without mux.
It should only be an error if we accepted mux and already discarded the RTCP ICE component.
